zebra-consensus/src/primitives/ed25519.rs

```diff
@@ -0,0 +1,127 @@
+//! Async Ed25519 batch verifier service
+
+#[cfg(test)]
+mod tests;
+
```

Comment:
```suggestion
```
Not needed.

Reply:
This code imports the new `ed25519::tests` module when we're testing. Otherwise the tests don't run.

Reply:
Sorry about that, i misunderstood the code there. Thanks.

Reply:
No worries!